### DISCUSSION TITLE
val3 driver for ROS2

### DISCUSSION THREAD

**Discussion by maxkremmel** [ros-industrial/staubli_val3_driver#53] [General]
Hey there!
Short question: is it planned to maker a ROS2 version for the val3 driver package?

**Comment by gavanderhoorn** [comment]
There are no immediate plans, no.

But there also wouldn't really be anything to port to ROS 2 *here*, in this repository.

This repository only hosts the server side. The client (what you'd probably normally call a "driver") is provided by `ros-industrial/industrial_core`.
